db/migrate/20171221143918_drop_intra_alaska_base_linehauls.rb

```diff
@@ -0,0 +1,12 @@
+class DropIntraAlaskaBaseLinehauls < ActiveRecord::Migration[5.1]
+  def change
+    drop_table :intra_alaska_base_linehauls do |t|
+      t.int4range :dist_mi, null: false
+      t.int4range :weight_lbs, null: false
+      t.integer :rate, null: false
+      t.daterange :effective, null: false
+
+      t.timestamps null: false
+    end
```

Comment:
I don't think you need to pass a block to `drop_table`. `drop_table :intra_alaska_base_linehauls` should do the trick.

Reply:
I was following the recommendation from here https://stackoverflow.com/a/31657066/501015